tests/canary_image_checks.py

```
@@ -0,0 +1,30 @@
+"""Canary file to verify PR Container Image Checks workflow.
+
+Each variable below triggers (or suppresses) a specific image check rule.
+Remove this file once the CI workflow is validated.
+"""
+
+# IMG001: stray image not in any constants class (should warn)
+CANARY_STRAY = "quay.io/canary/stray-image:v1"
+
+# IMG001 suppressed: stray image, check suppressed
+CANARY_STRAY_SUPPRESSED = "quay.io/canary/stray-suppressed:v1"  # noqa: IMG001
```

Ruff (0.15.21)

[warning] Invalid rule code in `# noqa`: IMG001

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)